app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
```

Comment:
Do not forget to separate different groups of imports with a blank line according to PEP-8.